packages/roosterjs-content-model-dom/lib/domToModel/processors/childProcessor.ts

```diff
@@ -12,19 +12,21 @@ import type {
  * @param group The parent block group
  * @param parent Parent DOM node to process
  * @param context DOM to Content Model context
+ * @param processNonVisibleElements If true elements that has display:none style will be processed @default false
  */
-export const childProcessor: ElementProcessor<ParentNode> = (
+export const childProcessor: ElementProcessor<ParentNode, boolean> = (
     group: ContentModelBlockGroup,
     parent: ParentNode,
-    context: DomToModelContext
+    context: DomToModelContext,
+    processNonVisibleElements: boolean = false
```

Comment:
No need to add a new parameter, but you can integrate the parameter into `context`